### PR TITLE
[17.0][MIG] sale_sourced_by_line

### PR DESCRIPTION
**Pull request by celm1990**
Depends on:
sale_procurement_group_by_line: https://github.com/OCA/sale-workflow/pull/2915

### CODE REVIEW

File: sale_sourced_by_line/tests/test_sale_sourced_by_line.py
Comment (rousseldenis) [inline sale_sourced_by_line/tests/test_sale_sourced_by_line.py]: Could you change to classmethod ?

File: sale_sourced_by_line/tests/test_sale_sourced_by_line.py
Comment (rousseldenis) [inline sale_sourced_by_line/tests/test_sale_sourced_by_line.py]: FYI, a better test base class is : https://github.com/odoo/odoo/blob/140e58c5a68db674fc1d240147e4d669b29f2cad/odoo/addons/base/tests/common.py#L30

in order to include performances enhancement (DISABLED_MAIL_CONTEXT)

**Review by rousseldenis — CHANGES_REQUESTED** [review]
A little change

File: sale_sourced_by_line/tests/test_sale_sourced_by_line.py
Comment (celm1990) [reply sale_sourced_by_line/tests/test_sale_sourced_by_line.py]: done

File: sale_sourced_by_line/tests/test_sale_sourced_by_line.py
Comment (celm1990) [reply sale_sourced_by_line/tests/test_sale_sourced_by_line.py]: thanks, done

**Review by Fabiola-Auz — APPROVED** [review]

**Comment by celm1990** [comment]
@OCA/crm-sales-marketing-maintainers could somebody help me with review or merge, please

**Comment by codeagencybe** [comment]
When is this one going to merge?

**Comment by celm1990** [comment]
ping @OCA/crm-sales-marketing-maintainers

**Comment by celm1990** [comment]
This PR has been waiting for a long time. Can someone help me? Ping @OCA/crm-sales-marketing-maintainers.
@rousseldenis, could you please update your review?

CC: @pedrobaeza

**Comment by pedrobaeza** [comment]
I think you should rebase for having a functional runboat, but @rousseldenis must unblock this.

**Review by rousseldenis — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@celm1990 please check CI

**Comment by celm1990** [comment]
> @celm1990 please check CI

@pedrobaeza  fixed in https://github.com/OCA/sale-workflow/commit/963f411a9b884b8909a64a07281fcbd961f627ab

**Pull request merged**
